web-ui/src/helpers/datetime.js

```diff
@@ -0,0 +1,11 @@
+import { startOfQuarter, endOfQuarter } from 'date-fns';
+
+/**
+ * Returns the start and end dates of the quarter that the given date falls in.
+ * @param {Date} inputDate The date to get the quarter duration for.
+ * @returns {{ startOfQuarter: Date, endOfQuarter: Date }} The start and end dates of the quarter.
+ */
+export const getQuarterDuration = inputDate => ({
```

Comment:
"Duration" doesn't seem like the right word to describe what this does.
Maybe `getQuarterBeginEnd` would be better.

Reply:
That's fair. Updating...